site/_includes/layout.njk

```diff
@@ -8,6 +8,7 @@ siteTitle: Web features explorer
     <title>{{ siteTitle }} - {{ title }}</title>
     <meta name="viewport" content="width=device-width, initial-scale=1">
     <link rel="stylesheet" type="text/css" href="/assets/styles.css">
+    <link rel="alternate" type="application/rss+xml" title="Web platform release notes" href="/release-notes.xml">
```

Comment:
```suggestion
    <link rel="alternate" type="application/atom+xml" title="Web platform release notes" href="/release-notes.xml">
```

The feed you're trying to add is technically an Atom feed so this should use atom+xml instead of rss+xml